src/utils/BlobDirection.ts

```diff
@@ -0,0 +1,35 @@
+export type BlobDirection = 'none' | 'up' | 'down'
+export interface BaseBlobProps {
+  direction: BlobDirection
+}
+
+const blobDirectionStyles = {
+  none: {
+    top: 0,
+    display: 'none',
+  },
+  up: {
+    top: 'auto',
+    bottom: 0,
+  },
+  down: {
+    top: 0,
+    bottom: 'auto',
+  },
+}
+
+const upBlobPathData: string =
```

Comment:
not that it's bad to have it, but i think TS can infer the `string` type :)

Reply:
True!